README.md

```diff
@@ -15,6 +15,7 @@ Setup
 * Install Vagrant and Virtualbox
 * Initialize the vagrant box - `vagrant up`
 * Log into the server - `vagrant ssh`
+* Load fixture data `./manage.py loaddata "core/fixtures/core-test.json"` (optional)
```

Comment:
You might want to spell out this this creates sample data, as "optional" isn't very verbose.